.env.example

```
@@ -13,7 +13,7 @@ PUB_API_BASE_URL=
 OS_BASE_URL=
 SCOPES_BASE_REALM=${API_BASE_URL}
 PURCHASES_API_URL=https://purchases-api.dev.fnopen.com
-PURCHASES_API_SCOPES="purchases-show-medata/read purchases-show-medata/write show-form/read show-form/write customized-form/write customized-form/read carts/read carts/write purchases/read cart-notes/write"
+PURCHASES_API_SCOPES="purchases-show-medata/read purchases-show-medata/write show-form/read show-form/write customized-form/write customized-form/read carts/read carts/write purchases/read cart-notes/write payment-profile/read"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PURCHASES_API_SCOPES key should go before the PURCHASES_API_URL key

(UnorderedKey)